aggregate-snapshot/aggregate-snapshot-service/src/main/java/uk/gov/justice/services/eventsourcing/source/core/SnapshotAwareEventSource.java

```diff
@@ -34,6 +39,13 @@ public EventStream getStreamById(final UUID streamId) {
         return new SnapshotAwareEnvelopeEventStream(streamId, eventStreamManager, snapshotService);
     }
 
+    @Override
+    public Stream<EventStream> getEventStreamsBySequence(final long sequenceNumber) {
```

Comment:
this naming is not clear.

BySequence suggests you'd get a single EventStream. Why is it returning a Stream>

Reply:
EventSource currently supports 2 different methods 
1.  Getting an EventStream - getStreamById
2. Getting a Stream<EventStream> --> getEventStreamsBySequence(SequenceNumber)

Sot getEventStreamsBySequence(SequenceNumber) means get  e a stream of Event log entries for a particular sequence in the Event Stream tabele

Reply:
See my review on the framework-api